controller-manager/src/main/resources/templates/PrometheusRules-enmasse.yaml

```diff
@@ -20,6 +20,9 @@ spec:
     - record: enmasse_component_health
       expr: up{job="address-space-controller"} or on(namespace) (1- absent(up{job="address-space-controller"}) )
 
+    - record: enmasse_component_health
+      expr: up{job="job="enmasse-operator-metrics""} or on(namespace) (1- absent(up{job="job="enmasse-operator-metrics""}) )
```

Comment:
My bad! I thought I was the indentation causing the error but I just noticed there's another typo. `job="` is in the labels twice